api/mutations/analytics/addPageView.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import { parse as parseURL } from 'url';
```

Comment:
Minor nitpick: `url.parse` is deprecated, it's recommended to use the `URL` constructor now: https://nodejs.org/api/url.html#url_constructor_new_url_input_base

```JS
import { URL } from 'url';

// ...

const parsedUrl = new URL(url);
```

Reply:
oh, didn't know that it was deprecated.
Thanks for that, I've updated the code.